lib/vagrant-service-manager/service.rb

```diff
@@ -0,0 +1,35 @@
+# Loads all services
+Dir["#{File.dirname(__FILE__)}/services/*.rb"].each { |f| require_relative f }
+
+module Vagrant
+  module ServiceManager
+    SUPPORTED_BOXES = ['adb', 'cdk']
+
+    class Service
+      def initialize(app, env)
+        @app = app
+        @machine = env[:machine]
+        @ui = env[:ui]
+        @services = @machine.config.servicemanager.services.split(',').map(&:strip)
+        @docker_hook = Docker.new(@machine, @ui)
+        @openshift_hook = OpenShift.new(@machine, @ui)
+      end
+
+      def call(env)
+        @app.call(env)
+
+        if SUPPORTED_BOXES.include? @machine.guest.capability(:os_variant)
+          @docker_hook.execute
+
+          if @machine.guest.capability(:os_variant) == "cdk"
```

Comment:
@bexelbie Does it mean that the line in Vagrantfile drvies the execution of openshift? 

I assume this:
1. If no service line -->  Execute only docker
2. If service line with "openshift" --->  Execute docker and openshift
3. If service line with "docker, openshift" ---> Execute docker and openshift

Similar for kubernetes.
So here variant will not decide which service to start.
Right?